.mvn/maven.config

```diff
@@ -1,2 +1,3 @@
 --strict-checksums
-
+-bsmart
```

Comment:
There's no noticable difference when using this plugin, maybe we can just keep `-T2C` here?